release/release_tests.yaml

```diff
@@ -1707,21 +1707,31 @@
     wait_for_nodes:
       num_nodes: 3
 
-
-  smoke_test:
-    frequency: nightly
-
-    run:
-      timeout: 3600
-
   alert: long_running_tests
 
+  variations:
+    - __suffix__: aws
+    - __suffix__: smoke-test
+      frequency: nightly
+      run:
+        timeout: 3600
+        script: python workloads/apex.py
+        long_running: true
+        wait_for_nodes:
+          num_nodes: 3
+    - __suffix__: gce
+      env: gce
+      frequency: manual
```

Comment:
[1] add this under this variation
                  
               smoke_test
                   frequency: manual
                   run:
                         timeout: 3600